chore(release notes)Cloud release notes cycle37 (#1040)
seqeralabs/docs: changelog/seqera-cloud/26.1.0_cycle37.md

---
title: Seqera Cloud v26.1.0_cycle37
date: 2026-01-14
tags: [seqera cloud]
---

## General updates and improvements

### Studios

  - Added validation to ensure the workspace container repository setting is configured before allowing custom-built Studios to be created.
  - Added Studios to the onboarding resources shown to new users during account setup, helping users discover Studio functionality early in their journey.
  - Removed the ability to start and stop Studios from the **Launch** role to better align permissions with user responsibilities.

### Compute

  - Added Amazon EC2 p6-b300 instance family to the NVMe storage list, enabling support for this new GPU-optimized instance type.
  - Implemented the ability to disable and enable compute environments through the UI. This allows administrators or workspace owners to temporarily or permanently prevent new workflow executions or new Studio sessions on selected compute environment(s).

>   - Improved error messaging when attempting to launch workflows on disabled compute environments. The error message now explicitly states **Specified compute environment has been disabled** instead of the generic **Not available** message.

Author: Another option is to reword it as: "Introduced error messaging when attempting to launch workflows on disabled compute environments. The error message now explicitly states **Specified compute environment has been disabled** instead of the generic **Not available** message."

 - Added tracking for Spot instance interruptions (`numSpotInterruptions`) at the task level, aggregated to the workflow level. This metric tracks how many times tasks were interrupted and restarted due to Spot instance preemption.

### Launchpad

 - Renamed the **Config view** button to **Params file view** in the launch form parameter section for better clarity and consistency with terminology.

## Bug fixes

### Studios

- Fixed an issue where basic organizations could run more than one Studio simultaneously. Basic organizations now default to a maximum of 1 running Studio when no explicit quota is set, while other organization types use the global limit (default: 100).

### Compute

- Fixed intermittent workflow launch failures caused by expired AWS STS credentials being returned from cache. Added expiration checking, configurable refresh margin, and cache invalidation for expired entries.
- Reverted Azure Batch to use Ubuntu 22.04 (from 24.04).
- Fixed errors when listing compute environments with the `resources` attribute for legacy Seqera Compute Batch configurations that don't implement `IAwsCloudConfig`. Added proper handling and fallback logic.

### Pipeline versioning

> - Fixed an issue where the pipeline versions list was not correctly updated when a draft version was published.

Reviewer: ```suggestion
- Fixed an issue with pipeline version lists not updating correctly when a draft version is published.
```

### Launchpad

> - Fixed an issue where parameters were not properly cleared when switching between pipelines with and without parameters in the launch form.

Reviewer: ```suggestion
- Fixed an issue with parameters not clearing correctly when switching between pipelines with and without parameters in the launch form.
```
